New Red Blood Bro hud icon

TeomanTheGreat opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
The color of the syndicate is red. The BB hud icon is pink. Now it's red.
New: 
![image](https://user-images.githubusercontent.com/55861563/96327984-5d8b6300-100c-11eb-962f-497566fa002b.png)

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
The color of the syndicate is red so the color of the BB hud icon should be too.
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl: Phil Smith
tweak: changed the color of the blood bro hud icon 
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

TeomanTheGreat committed:
New BB icon

eeSPee commented:
:cl:
imageadd: Added new red Blood Bro icon
imagedel: Deleted old Blood Bro icon
:cl:

to 

:cl:
tweak: changed the color of blood brother icon
:cl:

No?

TeomanTheGreat commented:
> :cl:
> imageadd: Added new red Blood Bro icon
> imagedel: Deleted old Blood Bro icon
> :cl:
> 
> to 
> 
> :cl:
> tweak: changed the color of blood brother icon
> :cl:
> 
> No?

Just changed it now
